src/targets/gpu/include/migraphx/gpu/rocblas.hpp

```diff
@@ -26,8 +26,9 @@
 #include <migraphx/manage_ptr.hpp>
 #include <migraphx/config.hpp>
 // for development:
-#define ROCBLAS_BETA_FEATURES_API
+#ifdef ROCBLAS_BETA_FEATURES_API
 #define ROCBLAS_NO_DEPRECATED_WARNINGS
```

Comment:
Are you seeing deprecated warnings if you don't define this?

Reply:
The actual warning I see reads like this:  `"rocblas_gemm_ex_get_solutions is a beta feature and is subject to change in future releases"`

Reply:
You can set this inside Cmakelists.txt then along with `ROCBLAS_BETA_FEATURES_API`